index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <title>PortfolioPage</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo-placeholder"><a href="#">Rich</a></div>
+      <nav class="menu"><img src="./logos/list.png" alt="menu" /></nav>
+    </header>
+    <main>
+      <section class="container">
+        <h1 class="primary-text">I’m Richmond Glad to see you</h1>
+        <p class="content">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
```

Comment:
Again here you have not used the right color for the text in this paragraph. Kindly use the color provided in the Figma design template.